### PR TITLE
Require Node.js 18, move to ESM and named exports

### PR DESCRIPTION
**Pull request by yannbertrand**
Followed [ESM docs](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) and your work on other projects.

API Breaking changes:
- `const aperture = require('aperture')` → `import {recorder} from 'aperture'`
- `await aperture().startRecording()` → `await recorder.startRecording`
- `aperture.screens` → `import {screens} from 'aperture'`
- `aperture.audioDevices` → `import {audioDevices} from 'aperture'`
- `aperture.videoCodecs` → `import {videoCodecs} from 'aperture'`

`ava`, `file-type`, `xo` and `electron-util` last versions require Node 18 so I couldn't use them. Do you think it's worth upgrading to Node 18 now?

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -1,25 +1,28 @@
-'use strict';
-const fs = require('fs');
-const delay = require('delay');
-const aperture = require('.');
+import fs from 'node:fs';
+import timers from 'node:timers/promises';
+import {aperture, screens, audioDevices, videoCodecs} from './index.js';
```
Comment (sindresorhus) [inline example.js:3]:
```suggestion
import {recorder, screens, audioDevices, videoCodecs} from './index.js';
```

Would make more sense to name it `recorder`, since we're doing changes anyway.

Comment (yannbertrand) [reply example.js:3]: What about exporting a `Recorder` class? Not sure about the demos example naming

```js
const recorder = new Recorder()
```

Comment (sindresorhus) [reply example.js:3]: You cannot record two things at once, so I don't think being able to create a new instance is very useful.

Comment (yannbertrand) [reply example.js]: I exported a `Recorder` instance named `recorder`.

File: readme.md [hunk 2]
```diff
@@ -8,63 +8,67 @@
 npm install aperture
 ```
 
-*Requires macOS 10.13 or later.*
+_Requires macOS 10.13 or later._
```
Comment (sindresorhus) [inline readme.md:11]: Don't make unrelated changes.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sindresorhus** [comment]
Thank you 🙏
